tests/stress/stress_test.py

```diff
@@ -326,6 +366,9 @@ def main():
 
     if args.heavy:
         print('Setting the heavy configuration...')
+        # Set the sizes of the large files to be bigger than the max memory on the system to test that data
+        # is being streamed when the large bundles are used as a dependencies.
+        args.large_dependency_size_gb = 16
```

Comment:
Do we have somewhere specifying the max size of memory? If yes, I think we could probably not hard-code here.

Reply:
No, there isn't an easy way to get that number.